using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -446,24 +510,30 @@ Identity.getUrlVariables(new AdobeCallback<String>() {
 {% tab title="iOS" %}
 ### getUrlVariables <a id="geturlvariables-ios"></a>
 
-_added in ACPCore v2.3.0 \(ACPIdentity v2.1.0\)_
+{% hint style="info" %}
+This method was added in ACPCore version 2.3.0 and ACPIdentity version 2.1.0.
+{% endhint %}
 
-Retrieve Adobe visitor data as a URL query parameter string for consumption in hybrid mobile applications. There is no leading "?" or "&" punctuation, as the caller is responsible for placing the string in the correct location of their resulting URL. The following information is added to the string that is returned via the callback:
+* This `getUrlVariables()` API gets the Visitor ID Service variables in URL query parameter form for consumption in hybrid app. This method will return an appropriately formed string containing Visitor ID Service URL variables. There will be no leading (&) or (?) punctuation, as the caller is responsible for placing it in their resulting java.net.URI in the correct location.
```

Comment:
Please change to:
"* This API gets the Visitor ID Service variables in URL query parameter form, and these variables will be consumed in the hybrid app. This method returns an appropriately formed string that contains the Visitor ID Service URL variables. There will be no leading (&) or (?) punctuation, because the caller is responsible for placing the variables in their resulting `java.net.URI` file in the correct location."